pytorch_lightning/callbacks/gpu_stats_monitor.py

```diff
@@ -112,12 +112,18 @@ def on_train_start(self, trainer, pl_module):
 
         self._gpu_ids = ','.join(map(str, trainer.data_parallel_device_ids))
 
-    def on_train_epoch_start(self, trainer, pl_module):
+    def on_train_epoch_start(self, *args, **kwargs):
```

Comment:
how this replacement would align if a user subclass this Monitor, does he can still write 
```
def on_train_epoch_start(self, trainer, pl_module):
    ...
```

Reply:
yes, overriding will still work.